public/app/percona/shared/components/Elements/Icons/Ellipsis.styles.ts

```diff
@@ -0,0 +1,25 @@
+import { css, keyframes } from 'emotion';
+
+const bounce = keyframes`
```

Comment:
Could you please share a video/GIF of how this looks?

Reply:
Sure, done 👍

Reply:
Great stuff! Will probably use it in DBaaS!